test/e2e-go/features/catchup/basicCatchup_test.go

```diff
@@ -78,12 +78,6 @@ func TestCatchupOverGossip(t *testing.T) {
 	t.Parallel()
 	// ledger node upgraded version, fetcher node upgraded version
 	runCatchupOverGossip(t, false, false)
-	// ledger node older version, fetcher node upgraded version
-	runCatchupOverGossip(t, true, false)
-	// ledger node upgraded older version, fetcher node older version
-	runCatchupOverGossip(t, false, true)
-	// ledger node older version, fetcher node older version
-	runCatchupOverGossip(t, true, true)
```

Comment:
I think that I have a better proposal for this test - 
improve it so that it would know how to read the list of SupportedProtocolVersions and dynamically use these.

The motivation here is that I expect to have another network version soon, and this test seems to be a good test case for that.

Reply:
If the list contains just a single protocol then we can just execute runCatchupOverGossip with (false, false)